addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -87,6 +87,21 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
             sequence: options.sequence || 1000,
             test: options.test,
             wait_for: options.wait_for || Promise.resolve(),
+            beyondReady: function(delay) {
+                // can be used by tests to allow chrome to continue making requests after tour last tour step
+                switch (tour.beyondReadyState) {
+                    case "started":
+                        return false;
+                    case "done":
+                        return true;
+                    default:
+                        tour.beyondReadyState = "started";
+                        setTimeout(function() {
+                            tour.beyondReadyState = "done";
+                        }, delay || 1000);
+                        return false;
+                }
+            },
```

Comment:
Sorry I don't understand the point of this commit... you are speaking of "the test cursor was closed right after the
last step of the tour was completed" but this commit seems to change the opposite by delaying the first step of the tour... which is already the point of the "wait_for" property.

Can you re-explain it to me?